tests/files/debian11-kubelet-csr-approver.yml

```diff
@@ -6,5 +6,9 @@ cloud_image: debian-11
 kubelet_rotate_server_certificates: true
 kubelet_csr_approver_enabled: true
 kubelet_csr_approver_values:
-  # Do not check DNS resolution in testing (not recommended in production)
-  bypassDnsResolution: true
+
+# Do not check DNS resolution in testing (not recommended in production)
+bypassDnsResolution: true
```

Comment:
Why this delete the indent?

Reply:
Thanks. 
Nice catch! I thought it was a bug and fixed it, but it turned out I was mistaken. 😓